components/badge/badge.md

```diff
@@ -1,57 +1,35 @@
 Position top:
 
-    <Badge position="top" title="Badge">
-      <div style={{border: '1px solid lightgray', height: '70px', marginTop: '15px'}} />
-    </Badge>
-
-Position right:
-
-    <Badge position="right" title="Badge">
-      <div style={{border: '1px solid lightgray', height: '70px', marginTop: '15px'}} />
-    </Badge>
-
-Position right with arrow:
-
-    <Badge arrow position="right" title="Badge">
-      <div style={{border: '1px solid lightgray', height: '70px', marginTop: '15px'}} />
-    </Badge>
-
-Position bottom:
-
-    <Badge position="bottom" title="Badge">
-      <div style={{border: '1px solid lightgray', height: '70px', marginTop: '15px'}} />
-    </Badge>
-
-Position left:
-
-    <Badge position="left" title="Badge">
-      <div style={{border: '1px solid lightgray', height: '70px', marginTop: '15px'}} />
-    </Badge>
-
-Position left with arrow:
-
-    <Badge arrow position="left" title="Badge">
-      <div style={{border: '1px solid lightgray', height: '70px', marginTop: '15px'}} />
-    </Badge>
-
-Without title:
-
-    <Badge>
-      <div style={{border: '1px solid lightgray', height: '70px', marginTop: '15px'}} />
-    </Badge>
-
-Without content:
-
-    <Badge title="Badge" />
-
-Without content and with right arrow:
-
-    <Badge arrow position="right" title="Badge" />
-
-Without content and with left arrow:
-
-    <Badge arrow position="left" title="Badge" />
-
-Without border:
-
-    <Badge border={false} title="Badge" />
+    initialState = {
+      isVisibleTop: false,
+      isVisibleRight: true,
+      isVisibleBottom: true,
+      isVisibleLeft: true,
+      isArrow: false,
+      isBorder: true,
+    };
+
+    <div>
+      <Checkbox name="isVisibleTop" checked={state.isVisibleTop} onChange={() => setState({ isVisibleTop: !state.isVisibleTop })}>top</Checkbox>
```

Comment:
What if reformat all those single lines components to multi lines?
```
<Checkbox
  name="isVisibleTop"
  checked={state.isVisibleTop}
  onChange={() => setState({ isVisibleTop: !state.isVisibleTop })}
>
  top
</Checkbox>
```
It's gonna be much easier to read and maintain I think...